### PR TITLE
Add German localization

### LINKED ISSUE #48: Add German localization
Request to add German localization.

I will propose a PR for German localization right away.

### PR DESCRIPTION
**Pull request by Maschina**

### CODE REVIEW

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Kurzbefehl aufnehmen";
+"press_shortcut" = "Kurzbefehl wählen...";
+"keyboard_shortcut_used_by_menu_item" = "Diese Tastaturkürzel kann nicht verwendet werden, da es bereits durch den Menüpunkt „%@” belegt ist.";
```
Comment (D4rthR4tz3) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:3]:
```suggestion
"keyboard_shortcut_used_by_menu_item" = "Dieses Tastaturkürzel kann nicht verwendet werden, da es bereits durch den Menüpunkt „%@” belegt ist.";
```

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcut_used_by_system" = "Dieses Tastaturkürzel kann nicht verwendet werden, da es systemweit bereits verwendet wird.";
```
Comment (D4rthR4tz3) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:4]:
```suggestion
"keyboard_shortcut_used_by_system" = "Dieses Tastaturkürzel kann nicht verwendet werden, da es bereits systemweit verwendet wird.";
```

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können in  „Systemeinstellungen › Tastatur › Kurzbefehle“.";
```
Comment (D4rthR4tz3) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:5]:
```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Tastaturkurzbefehle“ geändert werden.";
```

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (alexander-heimbuch) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:3]: It should be “Dieses”

**Review by D4rthR4tz3 — CHANGES_REQUESTED** [review]
Only some small changes are necessary. I've added some suggestions. 🙂

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings (hunk 3 above)
Comment (alexander-heimbuch) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:5]: There seems to be a part missing. What can be done in the system preferences?

Comment (rejas) [reply Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:5]: There is one space too much before "„Systemeinstellungen >

**Comment by robinscholz** [comment]
It should be … (single glyph) instead of ... (three glyphs)

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "Kurzbefehl aufnehmen";
+"press_shortcut" = "Kurzbefehl wählen...";
```
Comment (Maschina) [inline Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:2]: As proposed by @robinscholz 
```suggestion
"press_shortcut" = "Kurzbefehl wählen…";
```

File: Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings (hunk 3 above)
Comment (Maschina) [reply Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:5]: > "Die meisten systemweiten Tastaturkürzel können unter „Systemeinstellungen › Tastatur › Tastaturkurzbefehle“ geändert werden."

```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Kurzbefehle“ geändert werden.";
```

Comment (Maschina) [reply Sources/KeyboardShortcuts/Localization/de.lproj/Localizable.strings:5]:
```suggestion
"keyboard_shortcuts_can_be_changed" = "Die meisten systemweiten Tastaturkürzel können unter  „Systemeinstellungen › Tastatur › Kurzbefehle“ geändert werden.";
```

**Review by staabm — APPROVED** [review]

**Review by jdreesen — APPROVED** [review]

**Comment by sindresorhus** [comment]
Thanks, everyone :)

**Pull request merged**